examples/nodejs6/debug.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function (context, params) {
```

Comment:
This seems really useful. Do we really want to bury it in the examples?

Reply:
I'm not sure where else we could place it, would like this to be moved to tests directly?